numpyro/distributions/constraint_registry.py

```diff
@@ -0,0 +1,45 @@
+# The implementation follows the design in PyTorch: torch.distributions.constraint_registry.py
```

Comment:
I think it will be a good idea to also add PyTorch license/copyright to the top of these files.

Reply:
PyTorch licence includes [many copyrights](https://github.com/pytorch/pytorch/blob/master/LICENSE). Which one would you suggest?

Reply:
We can add the copyright for PyTorch, and the conditions towards the end, i.e. just leave out the caffe2 stuff.

Reply:
Yup, already did it!